addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -283,7 +283,7 @@
                                     <td name="account_invoice_line_name">
                                         <t t-if="line.product_id">
                                             <t t-set="english_name" t-value="line.with_context(lang='en_US').product_id.display_name"/>
-                                            <t t-set="arabic_name" t-value="line.with_context(lang='ar_001').product_id.display_name"/>
+                                            <t t-set="arabic_name" t-value="line.with_context(lang=o.env['res.lang']._get_code('ar_001')).product_id.display_name"/>
```

Comment:
else 'en_US'?

Reply:
Are you sure there are no other contexts where we do with_context(lang='ar_001') (maybe in l10n_sa?)  Are there dbs where en_US is not active?

Reply:
I confirm there are no other `with_context(lang='ar_001')`. 
I assume changing 'en_US' is not needed, there have never been an issue with it. My fix is mirroring the way it is done at the top of the [file](https://github.com/odoo/odoo/blob/18.0/addons/l10n_gcc_invoice/views/report_invoice.xml#L14-L15).